cpp/src/arrow/compute/kernels/scalar_compare.cc

```diff
@@ -772,6 +1107,16 @@ const FunctionDoc less_equal_doc{
     ("A null on either side emits a null comparison result."),
     {"x", "y"}};
 
+const FunctionDoc between_doc{
+    "Check if values are in the given range, val between a and b",
```

Comment:
This seems to say the same thing in two different ways.
Perhaps simply "Check if values are in the given range"?

Reply:
Ok.